wca-regulations.md

```diff
@@ -266,7 +266,7 @@ Note: Because Article and Regulation numbers are not reassigned when Regulations
 ## <article-10><solved-state><solvedstate> Article 10: Solved State
 
 - 10b) Only the resting state of the puzzle, after the timer has been stopped, is considered.
-- 10c) The puzzle may be in any orientation at the end of the attempt.
+- 10c) The puzzle may be in any orientation at the end of the solve.
```

Comment:
End of a solve is when you stop turning the puzzles.
End of a attempt is when you/a judge stop the timer.
They are slightly different.

Reply:
The end of the solve is when the timer stops, as defined in:
https://www.worldcubeassociation.org/regulations/#A6a
The end of the attempt is when the competitor signs, as defined in:
https://www.worldcubeassociation.org/regulations/#A7c

Reply:
I am sorry the definition from my memory is not correct.  Please let me confirm with the WRC internally.  I will get back to asap.